### ISSUE TITLE
 allow to add a comment to a cron-entry

### ISSUE THREAD

**Issue by woutervh** [reinout/z3c.recipe.usercrontab#2]
new feature-request

IMHO,  in a project with many cronjobs (step1, stpe2, step3 ...),
it would be very nice to be able to add comment to cronjobs

```
[mycronjob]
recipe = z3c.recipe.usercrontab
times = 0 12 * * *
command = echo hello
comment =  step 1: say "hello"
```

would generate:

```
# Generated by /apps/buildouts/... [mycronjob]
# step 1: echo hello
#0 * * * *   echo hello
# END /apps/buildouts/... [mycronjob]
```

**Comment by reinout** [comment]
I agree, this would definitively be nice to have.

Are you planning to make a pull request for this yourself? Otherwise I don't mind doing it myself, but it would be silly to both do it at the same time :-)

**Comment by woutervh** [comment]
I propose you take this one,  
I'll make a pull-request for #1

**Comment by reinout** [comment]
Can you take a look at #3?

**Comment by reinout** [comment]
I released 1.3, can you check if it works for you?
